CI/freebsd/01_install_dependencies.sh

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env bash
+
+##############################################################################
+# FreeBSD dependency management function
+##############################################################################
+#
+# This script file can be included in build scripts or run directly
+#
+##############################################################################
+
+# Halt on errors
+set -eE
+
+install_build-deps() {
+    shift
+    status "Install OBS build dependencies"
+    trap "caught_error 'install_build-deps'" ERR
+
+    sudo pkg install -U -y $@
+}
+
+install_obs-deps() {
+    shift
+    status "Install OBS dependencies"
+    trap "caught_error 'install_obs-deps'" ERR
+
+    sudo pkg install -U -y $@
+}
+
+install_qt-deps() {
+    shift
+    status "Install Qt dependencies"
+    trap "caught_error 'install_qt-deps'" ERR
+
+    sudo pkg install -U -y $@
+}
+
+install_plugin-deps() {
+    shift
+    status "Install plugin dependencies"
+    trap "caught_error 'install_plugin-deps'" ERR
+
+    sudo pkg install -U -y $@
+}
+
+install_dependencies() {
+    status "Set up apt"
+    trap "caught_error 'install_dependencies'" ERR
+
+    BUILD_DEPS=(
+        "build-deps cmake ninja pkgconf curl ccache"
+        "obs-deps ffmpeg libx264 mbedtls mesa-libs jansson lua52 luajit python37 libX11 xorgproto libxcb \
+         libXcomposite libXext libXfixes libXinerama libXrandr swig dbus jansson libICE libSM libsysinfo"
+        "qt-deps qt5-buildtools qt5-qmake qt5-imageformats qt5-core qt5-gui qt5-svg qt5-widgets qt5-x11extras qt5-xml"
```

Comment:
Should be able to remove qt5-x11extras.  Can probably also remove it from .cirrus.yml.

Reply:
Removed. Left cirrus alone for now.